astrolabe/cli.py

```diff
@@ -604,17 +607,23 @@ def delete_test_cluster(ctx, spec_test_file, workload_file, org_id, project_name
     cluster_name = get_cluster_name(
         get_test_name(spec_test_file, workload_file),
         cluster_name_salt)
+    msg = f"Deleting cluster {cluster_name} in project {project_name}..."
+    print(msg)
 
     # Step-2: delete the cluster.
+    client =ctx.obj.client
     organization = cmd.get_organization_by_id(
-        client=ctx.obj.client, org_id=org_id)
+        client=client, org_id=org_id)
     project = cmd.get_project(
-        client=ctx.obj.client, project_name=project_name, organization_id=organization.id)
+        client=client, project_name=project_name, organization_id=organization.id)
     if project:
         try:
-            ctx.obj.client.groups[project.id].clusters[cluster_name].delete()
-        except AtlasApiBaseError:
-            pass
+            client.groups[project.id].clusters[cluster_name].delete().data
```

Comment:
What does accessing `.data` do that's different than just calling `delete()` here?

Reply:
I had copied the pattern from another location.  My assumption is that that accessing the data triggers a validation.